frontend/src/services/objectService.ts

```diff
@@ -82,6 +83,16 @@ export default {
     return comsAxios().head(`${PATH}/${objectId}`);
   },
 
+  /**
+   * @function searchForPermissions
+   * Returns a list of permissions for the object
+   * @param {Object} params Optional object containing the data to filter against
+   * @returns {Promise} An axios response
+   */
+  searchForPermissions(params?: Object) {
+    return comsAxios().get(`${OBJECT_PERMISSION_PATH}`, { params: params });
+  },
+
```

Comment:
We should probably be making a new permissionService.ts for these sets of queries instead of nesting them with the existing related services IMHO. The constant declaration looks very janky in this context.